packages/admin/admin-date-time/src/datePicker/DatePicker.tsx

```diff
@@ -28,13 +28,14 @@ export const DatePicker = (inProps: DatePickerProps) => {
         onChange,
         value,
         formatDateOptions,
-        endAdornment,
         required,
         placeholder,
         monthsToShow,
         minDate = defaultMinDate,
         maxDate = defaultMaxDate,
         slotProps,
+        startAdornment,
```

Comment:
Adding `startAdornment` here causes it to no longer be included in `inputWithPopperProps`, therefore removing it from existing usages.